### PR TITLE
Improve SPRT remaining estimate

### PR DESCRIPTION
**Pull request by dubslow**
This estimate isn't very smart, but it's still better than current lol.
Intended to be later superseded by #1545.

This PR depends on the previous cleanup PR.

### CODE REVIEW

File: server/fishtest/util.py [hunk 1]
```diff
@@ -343,17 +343,25 @@ def is_active_sprt_ltc(run):
...
-        # checking randomly, half the expected games needs still to be done
-        remaining_games = expected_games / 2
-    else:
-        expected_games = run["args"]["num_games"]
-        remaining_games = max(0, expected_games - r["wins"] - r["losses"] - r["draws"])
-    game_secs = estimate_game_duration(run["args"]["tc"])
-    return game_secs * remaining_games * int(run["args"].get("threads", 1)) / (60 * 60)
+        average_games = 95000
+        llr = abs(run["args"]["sprt"]["llr"]) + 0.001
+        # Dumb estimate, very dumb near 0
```
Comment (ppigazzini) [inline server/fishtest/util.py:354]: Possible div by zero in the next two lines.

Comment (dubslow) [reply server/fishtest/util.py:352]: `rate` can't be zero because `llr` can't be zero, however yes `N` could be 0, oops

**Comment by ppigazzini** [comment]
@dubslow @MinetaS
time remaining 72h on PROD running with #1567 and #1568 
time remaining 82h on DEV running with #1545

**Comment by dubslow** [comment]
To be absolutely clear, this was only intended as a stopgap, an improvement on the fairly-bad-status-quo, until such time as 1545 is complete/consensus/etc.

**Comment by ppigazzini** [comment]
I don't have a baseline to judge if a estimate of the time remaining is acceptable, so I run the two proposals on the nearly same data: they seems to agree.

If @MinetaS agree I will merge his PR (that I already reviewed/tested for some days).

**Comment by MinetaS** [comment]
@ppigazzini I don't have enough time to review dubslow's PR, but I'm fine with any version if they lead to similar results.

**Comment by ppigazzini** [comment]
> @ppigazzini I don't have enough time to review dubslow's PR, but I'm fine with any version if they lead to similar results.

I was talking about your PR, from your last message I don't understand if your PR is finished, 
```
However I'm not sure about guessing remaining time based on LLR.
The metric is totally random on most of tests except decisive Elo gaining/losing patches.
```

**Comment by MinetaS** [comment]
#1545 is finalized. If there are any better ideas I'll make another PR for them.

**Comment by ppigazzini** [comment]
> #1545 is finalized. If there are any better ideas I'll make another PR for them.

Ok, thank you.

**Comment by ppigazzini** [comment]
Closed after merging #1545
Thank you @dubslow :)

**Pull request closed without merging**